internal/devserver/main/javascript.go

```diff
@@ -0,0 +1,3359 @@
+package main
+
+const RequireJs string = `/** vim: et:ts=4:sw=4:sts=4
```

Comment:
we can't copy their code into our repo for licensing reasons.
You can use a Bazel genrule to dynamically create `javascript.go` by copying files during the build.
Even better, can we read require.js from the runfiles and serve it off the disk instead?

Reply:
I thought the MIT license allows that as long as the copyright header is present?

```
const RequireJs string = `/** vim: et:ts=4:sw=4:sts=4
 * @license RequireJS 2.3.5 Copyright jQuery Foundation and other contributors.
 * Released under MIT license, https://github.com/requirejs/requirejs/blob/master/LICENSE
 */
```

Reply:
Maybe so - but I have to merge this change in google3 where we have more governance rules around third-party code (for example, any product that ships with this might need an end-user-visible list of licensed software it uses, and our tooling to verify this requires a certain layout)

Also it's messy :)

Reply:
Hmmm. Interesting problem. So you're thinking that the ts_devserver rule would take a list of scripts to include? Would work for RequireJs although would be let user friendly to always have to pull it from npm and include it in the rule. Would cause some trouble with livereloadJS. This script is modified based on the value of IBAZEL_LIVERELOAD_URL: `livereloadScript = fmt.Sprintf(LivereloadJs, livereloadScheme, livereloadHost, livereloadPort, livereloadPort)`. I took a similar approach to https://github.com/jaschaephraim/lrserver.

Reply:
No, I still think we should hard-code require.js is the one we want to load. But we probably should grab the version the user has installed (maybe it should be a peerDep of `@bazel/typescript`? or maybe we fix the version and make it a proper `dep` of `@bazel/typescript` and make sure we require our own version?)

Reply:
Ok. I think I can make it work for the livereload script as well now that I think about it. If this snippet is injected first then the vanilla livereload script should work: ```
  window.LiveReloadOptions = {
    https: true/false,
    host: theHost,
    port: thePort
  };
```